source/tests/utilities/test_helpers.h

```diff
@@ -1,21 +1,73 @@
 #ifndef TEST_HELPERS
 #define TEST_HELPERS
 
-#include <gtest/gtest.h>  // For EXPECT matchers
+#include <gmock/gmock.h>
+#include <gtest/gtest.h>
 
 #include "client_helpers.h"
 
-inline void expect_driver_error(const nidevice_grpc::experimental::client::grpc_driver_error& ex, int32_t expected_error)
```

Comment:
This broke the build because I added new calls to this function in #928. Fixed in #939